source/text/WrappedText.cpp

```diff
@@ -145,7 +145,7 @@ void WrappedText::Wrap(const char *str)
 /// With trailingBreak, include a paragraph break after the text.
 int WrappedText::Height(bool trailingBreak) const
 {
-	return height + trailingBreak * paragraphBreak;
+	return height + (height && trailingBreak) * paragraphBreak;
```

Comment:
```suggestion
	if(!height)
		return 0;
	return height + trailingBreak * paragraphBreak;
```
*Incredibly* pedantic, but I'd rather we be explicit in there being a special case for 0-height values.